README.md

```diff
@@ -299,6 +295,12 @@ There are a wide variety of networking issues that can occur while running `core
   `curl [URL]` (ipv4) and `curl -6 [URL]` (ipv6) from your shell.
 - Check your proxy settings (see [Environment Variables](#environment-variables)).
 
+## Non-goals
+
+### Official npm integration
+
+Official npm integration is a non-goal at this time. The npm team made it clear they don't wish npm to be distributed through Corepack (some of their concerns are listed [here](https://github.com/nodejs/node/issues/50963#issuecomment-1862957925)). Corepack focuses on the use case from officially supported package managers.
```

Comment:
That's no longer true since we added support for HTTP references, the goal is very much to help user run any package manager.